packages/itwinui-css/backstop/tests/information-panel.html

```diff
@@ -1310,7 +1318,8 @@ <h1>Information panel</h1>
               </label>
               <input
                 value="2021"
-                class="iui-input iui-small"
+                class="iui-input"
+                data-iui-size="small"
```

Comment:
all of that could have been done separately...

Reply:
I think these changes came from Jon's commits. This was a side effect of removing the component-height variables and aligning the logic with buttons.

Reply:
Cause then we have few more components, which also use class modifiers versus data attribute modifiers.. Sounds weird this change was required because of some variables change :/